File: backend/src/main/java/com/fl/dashboard/enums/TarefaStatus.java
package com.fl.dashboard.enums;

public enum TarefaStatus {
    BACKLOG("Backlog"),
    TODO("To Do"),
    IN_PROGRESS("In Progress"),
    IN_REVIEW("In Review"),
    DONE("Done");

    private final String displayName;

    TarefaStatus(String displayName) {
        this.displayName = displayName;
    }

    public String getDisplayName() {
        return displayName;
    }
}

File: frontweb/src/constants/tarefaStatus.ts
import { TarefaStatus } from '../types/tarefa';

export const TAREFA_STATUS_LABELS: Record<TarefaStatus, string> = {
  BACKLOG: 'Backlog',
  TODO: 'A Fazer',
  IN_PROGRESS: 'Em Progresso',
  IN_REVIEW: 'Em Revisão',
  DONE: 'Concluído',
};

export const getTarefaStatusLabel = (status: string): string =>
  TAREFA_STATUS_LABELS[status as TarefaStatus] ?? status;
